code/modules/holoparasite/_holoparasite.dm

```diff
@@ -229,14 +229,14 @@ GLOBAL_LIST_EMPTY_TYPED(holoparasites, /mob/living/simple_animal/hostile/holopar
 		. += "<span data-component=\"RadarChart\" data-width=\"300\" data-height=\"300\" data-area-color=\"[accent_color]\" data-axes=\"Damage,Defense,Speed,Potential,Range\" data-stages=\"1,2,3,4,5\" data-values=\"[stats.damage],[stats.defense],[stats.speed],[stats.potential],[stats.range]\" />"
 
 /mob/living/simple_animal/hostile/holoparasite/get_idcard(hand_first = TRUE)
-	//Check hands
+	// IMPORTANT: don't use ?. for these, because held_item might be 0 for some reason!!
```

Comment:
masking a bug. get_active_held_item() should never be returning a value that isnt null or an item instance

Reply:
Hmm, maybe that FALSE might have been a failsafe.
@Absolucy Would you post a testing evidence please?

Reply:
I don't even know what triggered the runtime, but sure in a bit

Reply:
@EvilDragonfiend here's it uh... not runtiming? 
![23-11-29-1701285068-dreamseeker](https://github.com/BeeStation/BeeStation-Hornet/assets/65794972/a09d7822-c7dd-44f2-a937-1981fcecae17)